cmd/fleet/calendar_cron.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/go-kit/log/level"
 )
 
+const calendarConsumers = 18
```

Comment:
curious why 18?  Does it have something to do with the googleapi rate limit?

Reply:
Yes, when deleting events, I'm seeing messages that we're hitting the rate limit. I think we should try to not hit the rate limit if possible. A 10% reduction seemed safe, considering the amount of QA already done.